securetea/args/arguments.py

```diff
@@ -398,5 +398,47 @@ def get_args():
         help="Start system log monitoring process"
     )
 
+    parser.add_argument(
+        '--server_log',
+        action="store_true",
+        required=False,
+        help="Start server log monitoring process"
+    )
+
+    parser.add_argument(
+        '--log_file',
```

Comment:
Can we provide a default option based on the log type?

Reply:
@rejahrehim I'm not sure if I understood the task properly, but if the log file path mapping is what we are looking for, then we have a dedicated mapping structure in our `engine.py`.

Here is glimpse of it:
```python
        # OS to log file path mapping
        self.system_log_file_map = {
                "apache": {
                    "debian": "/var/log/apache2/access.log​",
                    "fedora": "/var/log/httpd/access_log",
                    "freebsd": "/var/log/httpd-access.log"
                },
                "nginx": {
                    "debian": "/​var/log/nginx/access.log"
                }
        }
```
I have added the `log-file` argument if for any mean user wants to relocate the default log file. User can always leave this field blank and can just provide a log-type (apache/nginx), that's it. The path to the log file will be mapped accordingly based on their system.